main.c

```diff
@@ -1192,7 +1221,9 @@ void tick() {
               #if __EMSCRIPTEN__
                 Mix_HaltChannel(-1);
               #else
+		#if !defined(DISABLE_AUDIO)
```

Comment:
nit: Indentation is bad (tabs vs. spaces).
The indentation of the `Mix_FadeOutChannel` is also bad, but I'm not sure if I would touch it - touching more code means more conflicts where git doesn't know what to do, if 2 commits touched it.